modules/tfm/tfm/boards/board/tfm_peripherals_config.h

```diff
@@ -154,8 +154,14 @@ extern "C" {
 
 #define TFM_PERIPHERAL_VMC_SECURE            CONFIG_NRF_VMC_SECURE
 
-#if defined(CONFIG_NRF_GPIO1_PIN_MASK_SECURE)
+#if defined(CONFIG_NRF_GPIO0_PIN_MASK_SECURE)
```

Comment:
Could you take this bug-fix as a separate commit?

Reply:
sure